### DISCUSSION TITLE
Can the entry generated by the Duplicate operation be specified as draft state?

### DISCUSSION THREAD

**Discussion by blueMountain007** [payloadcms/payload#1268] [Q&A] [answered]
Can the entry generated by the Duplicate operation be specified as draft state?

<img width="406" alt="image" src="https://user-images.githubusercontent.com/16400882/196084656-cd7fc768-51d0-4374-bd0e-54237954ae84.png">

**Comment by DanRibbens** [comment, marked as answer]
Hi @zhounewz, this is now possible using the `beforeDuplicate` [admin hook](https://payloadcms.com/docs/configuration/collections#admin-hooks).

Here is an example of how this would look in your case:

```ts
const Posts: CollectionConfig = {
  slug: 'posts',
  admin: {
    hooks: {
      beforeDuplicate: ({ data }) => {
        return {
          ...data,
          _status: 'draft',
        };
      },
    },
  },
  versions: {
    drafts: true,
  },
  fields: [
    // ...
  ],
};
```

Cheers!
